### ISSUE TITLE
fix: missing focus indicator for checkbox

### ISSUE THREAD

**Issue by gahorn** [microsoft/fast-blazor#4629]
# 🐛 Bug Report

## 💻 Repro or Code Sample

```
@page "/"
@using Microsoft.FluentUI.AspNetCore.Components
<FluentStack Orientation="Orientation.Vertical" Padding="@Padding.All4">
    <FluentTextInput Label="Foo"/>
    <FluentCheckbox Label="Bar"/>
    <FluentCheckbox Label="Baz"/>
    <FluentRadioGroup TValue="string" Label="Qux">
        <FluentRadio Label="A" />
        <FluentRadio Label="B" />
        <FluentRadio Label="C" />
    </FluentRadioGroup>
    <FluentTextArea Label="Quux" />
</FluentStack>
```

## 🤔 Expected Behavior

checkbox shall have a visual cue (e.g. white border) when it has the input focus

## 😯 Current Behavior

no visual cue at all when the checkbox has input focus - other input components have it (as expected)

focus on first input field: visual cue is there
<img width="373" height="423" alt="Image" src="https://github.com/user-attachments/assets/c49904c7-ef90-4e9c-82ad-d0c2e3ee5c4b" />

--> TAB --> focus on checkbox "Bar": no visual cue
<img width="368" height="435" alt="Image" src="https://github.com/user-attachments/assets/8e291d99-ec26-47e4-a456-2ac89f11a4d6" />

--> TAB --> focus on checkbox "Baz": also no visual cue
<img width="369" height="435" alt="Image" src="https://github.com/user-attachments/assets/760ce49a-f43b-4e9b-9370-ae7d8b2e9b6c" />

--> TAB --> focus on radio group: we have a visual cue again
<img width="340" height="440" alt="Image" src="https://github.com/user-attachments/assets/315eb0a6-9d92-4f1d-9e79-dd3c9523bfc8" />

## 🌍 Your Environment

* OS & Device: Windows
* Browser: Microsoft Edge
* 5.0.0-preview.26074.1 (and various versions before that)

**Comment by vnbaaij** [comment]
We are investigating why it is not applying the :focus-visible correctly. In the mean time, adding the following to your CSS should help:

```
fluent-field:has(fluent-checkbox:focus-visible) {
  outline: -webkit-focus-ring-color auto 1px;
}
```
